options.h

```diff
@@ -45,6 +45,11 @@ bool        deliverLast = true;
 uint64_t    deliverSeq  = 0;
 bool        unsubscribe = false;
 
+uint64_t    interval         = 10000; // 10 seconds
+uint64_t    maxAttempts      = 1;
+uint64_t    reconnectBufSize = 1024; // 1 MB
```

Comment:
By default, the buffer size is 8Mb:
http://nats-io.github.io/nats.c/group__opts_group.html#gaa0d4b7ece8477cb9879f0dafff3456a5

But it can be very small for our usage, so set default buffer size to 8Mb and allow to change size to big values via options.